erpnext/selling/report/customer_wise_item_price/customer_wise_item_price.json

```diff
@@ -0,0 +1,43 @@
+{
+ "add_total_row": 0, 
```

Comment:
set :```"add_total_row": 0,```

Reply:
Why though? The sum of rates or stock is not meaningful.